src/Integration.Vsix/Suppression/LiveIssue.cs

```diff
@@ -34,15 +34,17 @@ public class LiveIssue
             string wholeLineText = "")
         {
             Diagnostic = diagnostic;
-            IssueFilePath = issueFilePath != string.Empty 
+            IssueFilePath = !string.IsNullOrEmpty(issueFilePath)
                 ? Path.GetFullPath(issueFilePath)
                 : string.Empty; 
             ProjectGuid = projectGuid;
             StartLine = startLine;
             WholeLineText = wholeLineText;
 
             // SonarQube doesn't calculate hash for file-level issues
-            LineHash = wholeLineText != "" ? ChecksumCalculator.Calculate(WholeLineText) : "";
+            LineHash = wholeLineText != string.Empty 
```

Comment:
Not something you've changed in this PR, but perhaps this logic should be inside the _ChecksumCalculator_.